android/src/main/java/com/reactnativecommunity/slider/ReactSliderManager.java

```diff
@@ -175,6 +176,13 @@ public void setMinimumTrackTintColor(ReactSlider view, Integer color) {
     }
   }
 
+  @ReactProp(name = "thumbImage")
+  public void setThumbImage(ReactSlider view, @Nullable ReadableMap source) {
+    if (source != null) {
+      view.setThumbImage(source.getString("uri"));
```

Comment:
what if we render the slider with `thumbImage={...}` and then re-render it with `thumbImage={undefined / null}`? Will the thumb image be removed?

Reply:
Yeah, you're right. The thumb is now reset if a `null` or `undefined` uri is supplied.